src/Makefile.manual

```diff
@@ -36,12 +42,12 @@ $(LIB): $(OBJS)
 	ar rcs $@ $(OBJS)
 
 clean:
-	$(RM) $(LIB) $(OBJS) $(MODS) $(SMODS)
+	$(RM) $(LIB) $(OBJS) $(MODS) $(SMODS) $(SRCGEN)
 
 %.o: %.f90
 	$(FC) $(FFLAGS) -c $<
 
-%.f90: %.fypp
+%.f90: %.fypp common.fypp
```

Comment:
Just a minor suggestion to introduce the correct scope for preprocessing:
```suggestion
$(SRCGEN): %.f90: %.fypp common.fypp
```
This avoids preprocessing a file from a `.fypp` which is only added as `.f90` in the `SRC` variable, but not present correctly in `SRCFYPP`.

Reply:
Thank you for the suggestion.